src/current/_includes/releases/v24.1/v24.1.4.md

```diff
@@ -0,0 +1,98 @@
+- Dropping [ENUM]({% link v24.1/enum.md %})-type values which were referenced by [index expressions]({% link v24.1/expression-indexes.md %}) could fail with an error. [#127454][#127454]
+- Fixed a bug that caused a memory leak when executing SQL statements with [comments]({% link v24.1/comment-on.md %}), for example, `SELECT /* comment */ 1;`. Memory owned by a SQL session would continue to grow as these types of statements were executed. The memory would only be released when closing the [SQL session]({% link v24.1/show-sessions.md %}). This bug has been present since v23.1. [#127759][#127759]
+- Addressed a bug where [schema changes]({% link v24.1/online-schema-changes.md %}) could hang if the lease rangefeed stopped receiving updates. [#127487][#127487]
+- Fixed small memory leaks that occur during [changefeed]({% link v24.1/change-data-capture-overview.md %}) creation. [#128018][#128018]
+- Fixed a memory leak that could occur when specifying a non-existent virtual cluster name in the connection string. [#128106][#128106]
+- Fixed a bug where [`CREATE INDEX IF NOT EXISTS`]({% link v24.1/create-index.md %}) would not correctly short-circuit if the given index already existed. [#128311][#128311]
+- Fixed a bug in syntax validation, in which the `DESCENDING` clause was not allowed for non-terminal columns of an [inverted index]({% link v24.1/inverted-indexes.md %}). Only the last column of an inverted index should be prevented from being `DESCENDING`. This is now properly checked. [#128311][#128311]
+- Fixed a bug where an [index]({% link v24.1/indexes.md %}) could store a column in the primary index if that column had a mixed-case name.  [#128311][#128311]
+- Setting or dropping a default value on a [computed column]({% link v24.1/computed-columns.md %}) is now blocked, even for `NULL` defaults. Previously, setting or dropping a default value on a computed column was a no-op. [#128466][#128466]
+- Fixed a bug that could cause spurious user permission errors when multiple databases shared a common schema with a routine referencing a table. The bug has existed since v22.2 when [user-defined functions (UDFs)]({% link v24.1/user-defined-functions.md %}) were introduced. [#126412][#126412]
+- Fixed a bug where [`debug zip`]({% link v24.1/cockroach-debug-zip.md %}) returns an error while fetching unstructured/malformed logs. [#128605][#128605]
+- Fixed a bug where a hash-sharded [constraint]({% link v24.1/constraints.md %}) could not be created if it referred to columns that had a backslash in the name. [#128521][#128521]
+- Fixed a bug in which output of [`EXPLAIN (OPT, REDACT)`]({% link v24.1/explain.md %}) of various `CREATE ...` statements was not redacted. This bug has existed since `EXPLAIN (REDACT)` was introduced in v23.1 and affects the statements `EXPLAIN (OPT, REDACT) CREATE TABLE`, `EXPLAIN (OPT, REDACT) CREATE VIEW`, and `EXPLAIN (OPT, REDACT) CREATE FUNCTION`. [#128489][#128489]
```

Comment:
```suggestion
- Fixed a bug in which the output of [`EXPLAIN (OPT, REDACT)`]({% link v24.1/explain.md %}) for various `CREATE` statements was not redacted. This bug had existed since [`EXPLAIN (REDACT)`]({% link v24.1/explain.md %}#parameters) was introduced in v23.1 and affects the following statements:
  - `EXPLAIN (OPT, REDACT) CREATE TABLE`
  - `EXPLAIN (OPT, REDACT) CREATE VIEW`
  - `EXPLAIN (OPT, REDACT) CREATE FUNCTION` [#128489][#128489]
```